### PR TITLE
Content page

### PR DESCRIPTION
**Pull request by DorienHuijser**
Made a start with 2 versions of the content page. The second one is similar to the speaker page, but is not divided into columns yet. I'm not sure how to fix that.

If we're good using the 2020 way, we can also use that :)

### CODE REVIEW

**Comment by DorienHuijser** [comment]
Edit: we also need to do something with the `linkname` in the yml file, that one is not used yet at the moment.  I suspect we may be able to use the linkname to link from the program page :)
Another edit: linkname is now integrated as div id for each card

**Comment by sofievdbos** [comment]
Looks greaaaaaaaaaaaaaaaat! ❤️ 
Thanks @DorienHuijser & @Remi-Gau!!!

**One remark**: 
We have three slide links on the program page & by now we probably have more that were shared with us. What about changing the "Slides" header on the program page to "Content" & link the content links to the respective content cards?

**Edit**: 
Checked the links, they all work! 🎉 Just saw that cuz Stephan has tooooo many links 🙈 the second "Repository" name doesn't fit in the box.

File: _data/content2021.yml [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    you recognise yourself to be in once in a while, you will notice that you will effortlessly vocalise
+    the lyrics of your favourite songs. Doesn’t that sound as a great opportunity for people to memorise 
+    your research better? Let’s put our papers in songs, hear all about it in this talk!"
+    materials: ~
+    video: "https://www.youtube.com/watch?v=p8Hh492Cb_k&list=PLav5iokutJ6rTGrkTcgGuSpTH7egT4xix&index=13"
+    repo: ~
+
+  - name: "Kayla Green"
+    linkname: "kayla"
+    image: "kayla.jpg"
```
Comment (sofievdbos) [inline _data/content2021.yml]:
```suggestion
    image: "kayla.JPG"
```

File: _data/content2021.yml [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+    do you get the most of Twitter, without it consuming all your time and energy? Based on a recent paper 
+    with the same name, in this talk I will discuss ten simple ideas that have helped me and other academics 
+    do exactly this."
+    materials: ~
+    video: "https://www.youtube.com/watch?v=JmoUUcisNpI&list=PLav5iokutJ6rTGrkTcgGuSpTH7egT4xix&index=16"
+    repo: ~
+
+  - name: "Moss Zhao & Patricia Clement"
+    linkname: "podcast"
+    image: "moss.jpg"
```
Comment (sofievdbos) [inline _data/content2021.yml]:
```suggestion
    image: {"moss.jpg","you.png"}
```

**Comment by DorienHuijser** [comment]
Agreed, let's change the "Slides" header to "Content" on the program page (I can do this later today)
"the second "Repository" name doesn't fit in the box." -> Mehh maybe change the name to Repo instead of Repository so that it fits? :D

**Comment by sofievdbos** [comment]
🚀 & 🚀 => sounds superrr!

**Comment by DorienHuijser** [comment]
Ok I think most things should work now, incl. the links from the program page to the content page. The only thing we need to do is now add the slides from speakers to the OSF page and then paste the links in the `content2021.yml` :D
Or am I still missing something else?

**Comment by sofievdbos** [comment]
You are so fast! Thought to surprise you by adding the links to the program page, but alas, you were faster. 😆
Can confirm that the links work! Sounds like the OSF is the only thing left indeed (except for maybe cross-checking titles on the content page with the ones on the program page, but that's a detail).

**Review by sofievdbos — APPROVED** [review]
LGTM! (first time I use this abbreviation 😁)

**Pull request merged**